Source/SpatialGDK/Public/SpatialActorChannel.h

```diff
@@ -128,6 +128,9 @@ class SPATIALGDK_API USpatialActorChannel : public UActorChannel
 	UPROPERTY(transient)
 	USpatialNetDriver* SpatialNetDriver;
 
+	worker::Map<worker::ComponentId, worker::InterestOverride> InterestOverrideMap;
+	bool bIsNetOwned;
+	bool bFirstReplication;
```

Comment:
This is kind of hacky, as I could not find a reliable flag to mark that the channel is new

Reply:
Well there is `bNetInitial` in `FReplicationFlags`. Not the worst hack.